samples/secrets-manager/infrastructure/.npmignore

```diff
@@ -0,0 +1,6 @@
+*.ts
```

Comment:
can we move this file to the root folder? Otherwise, we will end up haiving many .npmignore files